### PR TITLE
Added Floor Ignitor

### PR DESCRIPTION
**Pull request by Geekyhobo**
## About the PR

Do your burn chambers suck? Do you never use them? CE forget to give you the experimental welder? Want to just light gasses on fire in a chamber? Look no further, the Floor Based Ignitor will ignite your gasses, using the power of electricity. 

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added a Floor Ignitor to make your burn chamber experience better!

https://github.com/space-wizards/space-station-14/assets/66805063/706c2a1d-35e1-448f-8cc2-1bc83089e6c5

### CODE REVIEW

**Comment by deltanedas** [comment]
we already have heaters!!!!

**Comment by Geekyhobo** [comment]
Idk It just fills some niche use cases in Atmos

File: Resources/Textures/Structures/floor_ignitor.rsi/meta.json
Comment (Kit0vras) [inline Resources/Textures/Structures/floor_ignitor.rsi/meta.json:1]: change `; sprite has been edited` to `and modified by Geekyhobo (Github)`

Comment (Geekyhobo) [reply Resources/Textures/Structures/floor_ignitor.rsi/meta.json:1]: See newest commit

**Comment by Kit0vras** [comment]
maybe it would be better if it consumes electricity? it's an electrical device.

**Comment by Geekyhobo** [comment]
> maybe it would be better if it consumes electricity? it's an electrical device.

So, add apcpowerconsumer to it? how could i change the sprite when powered vs unpowered

**Comment by deltanedas** [comment]
theres a visualizer for power, make a layer and set it up like other machines such as a chem master

**Comment by Kit0vras** [comment]
> > maybe it would be better if it consumes electricity? it's an electrical device.
> 
> So, add apcpowerconsumer to it? how could i change the sprite when powered vs unpowered

> > maybe it would be better if it consumes electricity? it's an electrical device.
> 
> So, add apcpowerconsumer to it? how could i change the sprite when powered vs unpowered

For visualizer you can look at Prototypes/Entities/Structures/Machines, at example - telecomms, chem master

>apcpowerconsumer

  - type: ApcPowerReceiver 
     powerLoad: 50 
   - type: ExtensionCableReceiver

Or you can change parent to BaseMachinePowered

**Pull request closed without merging**